### DISCUSSION TITLE
Can't get it to work with traefik 3

### DISCUSSION THREAD

**Discussion by Weav3r** [tomMoulard/fail2ban#152] [Q&A] [answered]
I'm not able to get the plugin to work with traefik 3. Has anyone managed to do it here?
P.S. Thanks for this great plugin.

**Comment by Weav3r** [comment]
Asked too soon yet to try reply from maintainer in #139

**Comment by tomMoulard** [comment, marked as answer]
Hello @Weav3r,

What is your environment?

I have written a docker-compose file to test the plugin here : https://github.com/tomMoulard/fail2ban/blob/main/docker-compose.yml the fail2ban-register plug-in might be a good starting point for you

**Reply by Weav3r** [reply]
Thank you. Trying it now. I currently have it setup with traefik 2.11 and it does get started and identifies some (not all) malicious traffic but doesn't ban any of them. I know this is a skill issue, my traefik config is most likely the cause.
